packages/web-components/fast-components/src/card/index.ts

```diff
@@ -15,8 +27,80 @@ import { CardStyles as styles } from "./card.styles";
     name: "fast-card",
     template,
     styles,
+    shadowOptions: {
+        mode: "closed",
+    },
 })
-export class FASTCard extends Card {}
+export class FASTCard extends DesignSystemProvider
+    implements Pick<DesignSystem, "backgroundColor" | "neutralPalette"> {
+    /**
```

Comment:
because backgroundColor and neutralPalette aren't really supposed to be used by app authors, I would recommend flagging them as `@internal`